### PR TITLE
chore: add grid to event actions on eventPage

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

---

- Adds grid to event actions on eventPage to slightly unify the look with chapterPage.

### CODE REVIEW

File: client/src/modules/events/pages/eventPage.tsx [hunk 1]
```diff
@@ -353,72 +354,77 @@ export const EventPage: NextPage = () => {
...
-        {userEvent && (
-          <HStack>
-            {userEvent.subscribed ? (
-              <>
-                <Text fontSize={'md'} fontWeight={'500'}>
-                  You are subscribed to event updates
+          ) : (
+            <>
+              {attendanceStatus === 'waitlist' ? (
+                <Text fontSize="md" fontWeight="500">
```
Comment (Sboonny) [inline client/src/modules/events/pages/eventPage.tsx:373]: these styles are different from the other text, which weirding me out.

but this isn't related to the original issue, I am happy with current changes, if you think this deserves to be in another PR.

```suggestion
                <Text>
```

**Review by Sboonny — APPROVED** [review]

File: client/src/modules/events/pages/eventPage.tsx (hunk 1 above)
Comment (gikf) [reply client/src/modules/events/pages/eventPage.tsx:375]: I suppose there's couple more inconsistencies between the text - ie. text by subscribe/unsubscribe buttons have such styling, so I'd say separate PR would be better to get them all together. And to figure out whether the styling is needed or not.
